docs/ui/prepare-migration/dataflow-config.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Configure Dataflow (Optional)
+parent: Prepare Migration Page
+grand_parent: SMT UI
+nav_order: 4
+---
+
+# Configure Dataflow (Optional)
+{: .no_toc }
+
+In case of minimal downtime migration, the dataflow jobs launched by the Spanner Migration Tool can be **optionally** configured with custom runtime environment variables such as MaxWorkers, NumWorkers, specifying [networks and subnetworks](https://cloud.google.com/dataflow/docs/guides/specifying-networks) etc. 
+
+
+{: .highlight }
+The step shows configured (via a green tick) by default, denoting it is optional to configure.
+
+To configure dataflow, first specify the target database in the 'Configure Spanner Database' step. This enables the configure button for the remaining steps. The configurable fields for Dataflow are mentioned below.
+
+![](https://services.google.com/fh/files/misc/dataflow-config.png)
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+{: .highlight }
+Currently, we only allow configuring a subset of options available in Dataflow via SMT. For the full list of options available, click [here](https://cloud.google.com/dataflow/docs/reference/pipeline-options).
+
+## Host Project Id
+Specify the project id of the VPC that you want to use. This is required in order to use private connectivity. By default, this is assumed to be the same as Spanner project. 
```

Comment:
Let's mention what are the defaults, i.e. what happens if these fields are not provided

Reply:
Done